src/services/PeopleSearchService.ts

```diff
@@ -119,18 +119,17 @@ export default class SPPeopleSearchService {
           PrincipalSource: 15,
           PrincipalType: this.getSumOfPrincipalTypes(principalTypes),
           QueryString: query,
-          SharePointGroupID: null
```

Comment:
@c-eiser13 was this line causing issues, or are you just removing to revert to the previous working version?

Reply:
Ignore...just checked again the details of the related issue and I can see that this was indeed causing issues.